source/gui/exit.py

```diff
@@ -0,0 +1,163 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2006-2022 NV Access Limited, Peter Vágner, Aleksey Sadovoy, Mesar Hameed, Joseph Lee,
+# Thomas Stivers, Babbage B.V., Accessolutions, Julien Cochuyt, Cyrille Bougot
+# This file is covered by the GNU General Public License.
```

Comment:
Something I've been wondering about for some time now.  Why the copyright header used for new files is different from the one prescribed on the [copyright headers](https://github.com/nvaccess/nvda/wiki/Copyright-headers) wiki page? According to this page  this should be:
```
# This file may be used under the terms of the GNU General Public License, version 2 or later.
# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
```
Note that I'm happy either way just wanted to  point out that this is inconsistent with the documentation, and either this header or the wiki page should be modified not to introduce inconsistencies.

Reply:
@lukaszgo1 we try to remember to update them. It's very easy to copy from an open file. Ideally we'd have a lint rule for this, but we don't. On the other hand, it's not vitally important.

Reply:
```suggestion
# This file may be used under the terms of the GNU General Public License, version 2 or later.
# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
```